operator/molecule/deploy/role.yaml

```diff
@@ -0,0 +1,282 @@
+---
+# NOTE: Changes to this file must be reflected in the OperatorHub.io CSV file
```

Comment:
Is there something we can do here rather than hardcode the same role.yaml content that is found in the main operator/deploy directory? e.g., can we have a Makefile, and a "make test" will first copy the operator/deploy/role.yaml to some tmp directory and then the make target will run molecule with that file.

The same with the other files - looks like molecule/deploy just has copies of what is in operator/deploy (or at least a close copy - could probably be able to reuse them).

I really don't want to have to maintain another role.yaml (or other deploy yamls) - the ones in operator/deploy are authoritative and should be used if at all possible, rather than maintaining two separate copies (the "real" ones and the molecule ones).

Reply:
Yes, 
I think I will need to some sort of envbust parse.